pnpm-lock.yaml

```diff
@@ -1,5 +1,9 @@
 lockfileVersion: '6.0'
 
+settings:
```

Comment:
Interesting how Caleb's and Leo's latest changes to lock file pushed this section to the end of the file. Probably they use slightly older version of pnpm